config/locales/en.yml

```
@@ -78,6 +78,15 @@ en:
     claims_preventing_payment_message:
       one: This claim cannot currently be approved because we’re already paying another claim (%{claim_references}) to this claimant in this payroll month using different payment details. Please speak to a Grade 7.
       other: This claim cannot currently be approved because we’re already paying other claims (%{claim_references}) to this claimant in this payroll month using different payment details. Please speak to a Grade 7.
+    payment_deletion_claim_detail:
```

How about a more descriptive name for this?


```suggestion
    payment_deletion_claim_warning:
```